src/content/about-us/nossos-cooperados.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Nossos Cooperados"
```

Comment:
This is a list, I think will be more easy iterate on an json o yaml list of objects to generate the HTML.

Reply:
It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

Reply:
It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

Reply:
Is impossible mark all the document and I put the comment on first line.

Reply:
Thanks for the improvement, you're right, lists and json better work on this case.